src/lib/events/decisionsReceived/createDecisionsReceived.js

```diff
@@ -10,6 +10,10 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-module.exports = ({ decisionsCallbackStorage }) => (settings, trigger) => {
-  decisionsCallbackStorage.add(trigger);
+module.exports = ({ sendEventCallbackStorage }) => (settings, trigger) => {
+  sendEventCallbackStorage.add(result => {
+    if (result.decisions) {
+      trigger({ decisions: result.decisions });
+    }
+  });
 };
```

Comment:
I renamed this file to be consistent with what we've done elsewhere. That is, we don't suffix the files with `Event`, `Action`, `DataElement`, etc.